client/mapreduce/src/test/java/org/apache/accumulo/core/client/mapreduce/impl/BatchInputSplitTest.java

```diff
@@ -44,7 +44,7 @@
 
   @Test
   public void testSimpleWritable() throws IOException {
-    Range[] ranges = new Range[] {new Range(new Key("a"), new Key("b"))};
+    Range[] ranges = {new Range(new Key("a"), new Key("b"))};
     BatchInputSplit split = new BatchInputSplit("table", Table.ID.of("1"), Arrays.asList(ranges),
```

Comment:
This could be further simplified with `Collections.singletonList()` instead of allocating an intermediate array. I saw a few cases of this "array of singleton" case.

Reply:
Fixed in 96d0411ccda8fa3